README.md

```diff
@@ -137,8 +145,8 @@ By default, RedFlag produces an HTML report that can be opened in a browser.
 
 # CI Mode
 
-RedFlag can be run in CI pipelines to flag PRs and add the appropriate reviewers.
```

Comment:
Again this is more accurate as in this mode, RedFlag evaluates details of the PR (nothing more, nothing less).

Reply:
good feedback, I saw you committed so thanks! :)